pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -142,7 +142,7 @@ func getPipelineRunController(t *testing.T, d test.Data) (test.Assets, func()) {
 	ctx, cancel := context.WithCancel(ctx)
 	ensureConfigurationConfigMapsExist(&d)
 	c, informers := test.SeedTestData(t, ctx, d)
-	configMapWatcher := configmap.NewInformedWatcher(c.Kube, system.GetNamespace())
+	configMapWatcher := cminformer.NewInformedWatcher(c.Kube, system.GetNamespace())
```

Comment:
The point of this split btw was to enable you to write tests like this for your tiny containers:
https://github.com/mattmoor/mink/blob/master/cmd/extract-digest/depcheck_test.go#L27

Not necessary for this change, but we should follow up to add checks for all the containers that *should* be nice and tiny.